pkg/chainaccessor/default_accessor.go

```diff
@@ -749,6 +750,8 @@ func (l *DefaultAccessor) processCommitReports(
 	if len(reports) < limit {
 		return reports
 	}
+	lggr.Errorw("too many commit reports received, commit report results are truncated",
+		"numTruncatedReports", len(reports)-limit)
 	return reports[:limit]
```

Comment:
As I understand, this was source where the events got dropped.
Shouldn't we be atleast logging the reports that were dropped, as error logs?
What about a version similar to below, but of error type calling out these as dropped:
printTruncatedReports(lgger, reports[limit:])

Reply:
Good idea, adding additional logs for each report that doesn't make it to the result.